brambling/templates/brambling/event/organizer/create.html

```diff
@@ -0,0 +1,141 @@
+			</div>
+
+			<div class='row'>
+				<div class='col-sm-4'>
+					{% formrow form.start_date with prepend="<i class='fa fa-fw fa-calendar'></i>" %}
+				</div>
+				<div class='col-sm-4'>
+					{% formrow form.end_date with prepend="<i class='fa fa-fw fa-calendar'></i>" %}
+				</div>
+				<div class='col-sm-4'>
+					<label for='startEnd'>Add start / end times</label><br />
+					<input type='checkbox' id='startEnd'{% if event.start_time or event.end_time %} checked{% endif %}>
+				</div>
+			</div>
+			<div class='row' id='startEndRow'{% if not event.start_time and not event.end_time %} style="display:none;"{% endif %}>
+				<div class='col-sm-4'>
+					{% formrow form.start_time with prepend="<i class='fa fa-fw fa-clock-o'></i>" %}
+				</div>
+				<div class='col-sm-4'>
+					{% formrow form.end_time with prepend="<i class='fa fa-fw fa-clock-o'></i>" %}
+				</div>
+			</div>
+
+			{% with field=form.template_event %}
+				<div class='form-group {% if field.errors %} has-error{% endif %}'>
+					<label class='control-label' for="id_template_event">Template event <small class="text-muted">(Optional)</small></label>
+					<select class='chosen-select' id="id_template_event" name="template_event">
+						<option value=""></option>
+						{% with template_event_value=form|get_value:"template_event" %}
+							{% for event in field.field.queryset %}
+								<option value="{{ event.pk }}" data-org-id="{{ event.organization_id }}"{% if template_event_value == event.pk|stringformat:"i" %} selected{% endif %}>{{ event.name }}</option>
+							{% endfor %}
+						{% endwith %}
+					</select>
+					<p class='help-block'>If you choose a template event, all settings (including forms, discounts, and items) will be copied to your new event.</p>
+
+					{% if field.errors %}
+						{% include "floppyforms/errors.html" with errors=field.errors %}
+					{% endif %}
+				</div>
+			{% endwith %}
+		{% endform %}
+
+		<button class="btn btn-primary" type="submit">
+			Create event
+		</button>
+
+	</form>
+{% endblock %}
+
+{% block javascripts %}
+	{{ block.super }}
+	<script type="text/javascript">window.django = window.grp = {'jQuery': jQuery};</script>
+	<script type="text/javascript" src="{% static "admin/js/urlify.js" %}"></script>
+	<script type="text/javascript" src="{% static "admin/js/prepopulate.min.js" %}"></script>
+	<script type="text/javascript" src="{% static "brambling/brambling.slug.js" %}"></script>
+
+	{% include "brambling/event/organizer/_form_js.html" %}
+
+	<script>
+		var allEvents = $('#id_template_event').children(),
+			updateAvailableEvents = function() {
+				var orgId = $('#id_organization').val(),
+					$field = $('#id_template_event');
+				$field.children().remove();
+				allEvents.each(function (idx, ele) {
+					var $ele = $(ele);
+					if ($ele.attr('value') == '' || $ele.data('org-id') == orgId) {
+						$field.append($ele);
+					}
+				});
+				$field.trigger("chosen:updated");
+			};
+		updateAvailableEvents();
```

Comment:
By deleting and re-adding the available events here, it's breaking the "Copy Event" button. That is, when you visit the page with `template_event=N`, the event with id=N won't be initially selected because it's being removed from the select list and then added back (this time without the "selected" attribute).

One potential solution I see here is to just make the drop-down read-only if that GET argument is there. If they clicked the Copy Event button, then really they've already chosen what event they want as a template and don't need to mess with it anymore.